File: Programmers/Lv1/같은 숫자는 싫어.py
def solution(arr):
    answer = []
    tmp = []

    for w in arr:
        if len(tmp) <= 0:
            tmp.append(w)
        else:
            if tmp[-1] != w:
                answer.extend(tmp)
                tmp.clear()
                tmp.append(w)

    answer.extend(tmp)

    return answer

File: Programmers/Lv1/같은 숫자는 싫어.js
function solution(arr) {
  var answer = [];
  for (num of arr) {
    if (answer.length > 0) {
      let tmp = answer[answer.length - 1];
      if (tmp !== num) {
        answer.push(num);
      }
    } else {
      answer.push(num);
    }
  }
  return answer;
}
